contract/src/lib.rs

```diff
@@ -527,6 +529,7 @@ enum StorageKey {
     DropIdsForFunder,
     DropIdsForFunderInner { account_id_hash: CryptoHash },
     PksForDrop { account_id_hash: CryptoHash },
+    PasswordsPerUse { account_id_hash: CryptoHash },
```

Comment:
all of these are not "just" account_id_hash (they're hashes of something unique for that storage key. Should we name them to match what they are storing? It's kind of misleading.